int8        num_pois
float64[]   x
float64[]   y
